docs/config.toml

```diff
@@ -81,7 +81,7 @@ style = "tango"
 # Everything below this are Site Params
 
 [params]
-copyright = "The PipeCD Authors"
+copyright = "2020 - 2021 The PipeCD Authors"
```

Comment:
Using `now.Year` instead of the constant current year (2021) may be useful 👀 
In that case,
```diff
- copyright = "2020 - 2021 The PipeCD Authors"
+ copyright = "The PipeCD Authors"
+ since = 2020
```
then in the layout, we could use `{{ .Site.Params.since }}-{{ now.Year }}`

Reply:
Good idea. 👍

Reply:
That's nice!

Reply:
@khanhtc1202 @cakecatz Fixed it. Thank you.